test/core/formattersTests.ts

```diff
@@ -1,193 +1,280 @@
 ///<reference path="../testReference.ts" />
 
 describe("Formatters", () => {
-  describe("fixed", () => {
-    it("shows exactly [precision] digits", () => {
+  describe("fixed()", () => {
+    it("shows correct amount of digits according to precision", () => {
       let fixed3 = Plottable.Formatters.fixed();
```

Comment:
This is actually testing that the default formatter has precision 3. Either mention 3 or extract in a different it